postgres-appliance/build_scripts/base.sh

```diff
@@ -146,6 +146,22 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
         done
     )
 
+    if [ "${TIMESCALEDB_APACHE_ONLY}" != "true" ] && [ "${TIMESCALEDB_TOOLKIT}" = "true" ]; then
+        __versionCodename=$(sed </etc/os-release -ne 's/^VERSION_CODENAME=//p')
+        echo "deb [signed-by=/usr/share/keyrings/timescale_E7391C94080429FF.gpg] https://packagecloud.io/timescale/timescaledb/ubuntu/ ${__versionCodename} main" | tee /etc/apt/sources.list.d/timescaledb.list
+        curl -L https://packagecloud.io/timescale/timescaledb/gpgkey | apt-key --keyring /usr/share/keyrings/timescale_E7391C94080429FF.gpg add -
```

Comment:
```
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
```
can you please switch to `gpg --dearmor`

Reply:
Sure thing! Done :)